src/loaders.js

```diff
@@ -80,15 +85,16 @@ loaders.postcss.defaults = {
 	loader: require.resolve( 'postcss-loader' ),
 	options: {
 		postcssOptions: {
-			plugins: [
+			ident: 'postcss',
+			plugins: applyFilters( 'loader/postcss/plugins', [
 				postcssFlexbugsFixes,
-				postcssPresetEnv( {
+				postcssPresetEnv( applyFilters( 'loader/postcss/preset-env', {
 					autoprefixer: {
 						flexbox: 'no-2009',
```

Comment:
IMO we shouldn't be trying to auto-prefix or auto-anything CSS grid--using grid should be a per-project decision that comes down to "do we need to support IE or can we live in the year 2022 with the rest of the world?" My feeling is that one of the big benefits of grid is its (relatively) simple API and what you can accomplish with that, and although you can support _some_ stuff in IE though the use of vendor prefixed properties, etc, it creates an unreasonable mental load for engineers just to support an ancient browser that constitutes a sliver of global usage. The limitations imposed by the subset of features that IE (kind of) supports make it difficult to really _use_ grid, and IME it's frequently just easier to use flexbox in that situation, as it's better supported and with some cleverness you can get a lot of the same layout effects.

tl;dr:

- Supporting grid in IE 11 is not easily automateable
- IE has extremely low usage
- We should default to "IE is not supported"
- If IE support is a project requirement, then we should just forgo grid on that project
- Therefore we don't need autoprefixer to try and do anything with grid

Reply:
I'm inclined to agree with Ben because that feels both well-argued, and in line with what other boilerplates and standardized build processes do. Thank you for the opinion and the argument!